### PR TITLE
Vector of shared ptrs

### PR DESCRIPTION
**Pull request by BarTes8**
Authors: @BarTes8 , @MoreThanADC 
pair programming

### CODE REVIEW

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @BarTes8

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @MoreThanADC

**Comment by ziobron** [comment]
✅ CI Passed - Task: vector-of-shared-ptrs
🏅 Granted 5 XP for @BarTes8

**Comment by ziobron** [comment]
⏱ 3 XP extra for @BarTes8 for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: vector-of-shared-ptrs
🏅 Granted 5 XP for @MoreThanADC

**Comment by ziobron** [comment]
⏱ 3 XP extra for @MoreThanADC for on-time delivery

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    std::vector<std::shared_ptr<int>> vec;
+    vec.reserve(count);
+    for (int i = 0; i < count; i++) {
+        vec.push_back(std::make_shared<int>(i));
+    }
+    return vec;
+}
+
+void print(const std::vector<std::shared_ptr<int>>& vec) {
+    for (const auto& el:vec) {
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:13]: Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        if (el) {
+            std::cout << *el << "\n";
+        }
+    }
+}
+
+void add10(const std::vector<std::shared_ptr<int>>& vec) {
+    std::for_each(begin(vec), end(vec), [](auto& el) { if (el) {*el += 10; } });
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:21]: split it to 2 lines, will be more readable :)

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**